server/data/quotes.json

```diff
@@ -348,3 +348,67 @@
     "author": "William Butler Yeats"
   }
 ]
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+{
```

Comment:
This isn't valid JSON format. Please read [this](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md) guide first. Also, the quote you are adding is already a duplicate.